requirements.txt

```diff
@@ -0,0 +1,17 @@
+scipy
+pandas
+numpy
+matplotlib
+scikit_learn
+seaborn
+statsmodels
+jupyterlmod
+torch
+tensorflow
+tensorflow_probability
+pyyaml
+tqdm
+keras
+CEVAE_pytorch
+quince
+CompBioAndSimulated_Datasets
```

Comment:
To install the required packages, I used the 'pip install -r requirements.txt' command on Google Colab. Many of the packages were already installed on Colab, however since you didn't specify the package versions, Colab tries to find compatible versions and it took a while. I think specifying versions can boost this process. 
More importantly, pip couldn't install the three last packages 'CEVAE_pytorch', 'quince', and 'CompBioAndSimulated_Datasets'. In your  Jupyter file, you have commands for cloning the 'CompBioAndSimulated_Datasets', I used a similar command for cloning 'CEVAE_pytorch', and I could install these two packages. However, I still couldn't install 'quince'. Also, if using 'pip' is not suitable for installing these packages at all, please let me know. I think it would be great if you specifically mention how others can install these packages in .README file